tests/rptest/services/redpanda.py

```diff
@@ -39,6 +39,8 @@ class RedpandaService(Service):
     V_DEV_MOUNT = "/opt/v"
 
     # ducktape `--globals` for selecting build
+    BUILD_DIR_KEY = "redpanda_build_dir"
+    DEFAULT_BUILD_DIR = "vbuild"
```

Comment:
the default here is `vbuild`, but in ducker-rp i think it was `build`?

Reply:
Yes that is by design. The idea is if someone working with the open source build system, their default build dir will be `build` , so that is why `ducker-rp` defaults to `build`. In this case, vtools defaults to `vbuild` passing this to `ducker-rp`, because by using vbuild your probably in a scenario where your build dir is `vbuild`.